apps/server/lib/http/middlewares.ts

```diff
@@ -13,8 +18,10 @@ const logger = getLogger(__filename);
 export const attachMiddlewares = (
 	rootContext,
 	application,
-	_jellyfish,
-	options,
+	jellyfish: JellyfishKernel,
+	options: {
+		guestSessionId: string;
```

Comment:
Just curious, is this the only option ever passed to `attachMiddlewares()`?

Reply:
Yes, the `options` argument is only used once within the `attachMiddlewares()` function to access the `guestSessionId` attribute.